metagraph/src/graph/alignment/aligner_alignment.hpp

```diff
@@ -49,7 +58,13 @@ class Alignment {
               bool orientation = false,
               size_t offset = 0);
 
-    void append(Alignment&& other);
+    // Append other to the end of the current alignment. In this process, alignment
+    // labels are intersected. If coordinates are present, then the append is only
+    // successful if at least one coordinate of other immediately proceeds the
+    // one of the coordinates in this. If this operation is unsuccessful, then
+    // *this == {} afterwards.
+    // Returns true if the label or coordinate set of this changed.
+    bool append(Alignment&& other);
 
     size_t size() const { return nodes_.size(); }
     bool empty() const { return nodes_.empty(); }
```

Comment:
An example of confusing interface. `empty()` may return `false` even if all queues for all columns are empty.